.taskcluster.yml

```diff
@@ -10,302 +10,305 @@ tasks:
+                                                - index.mobile.v2.${project}.revision.${head_sha}.taskgraph.decision
+                                          - $if: 'tasks_for == "cron"'
+                                            then:
+                                                # cron context provides ${head_branch} as a short one
+                                                - index.mobile.v2.${project}.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
+                                                - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
+                                                - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
+                            scopes:
+                                $if: 'tasks_for == "github-push"'
+                                then:
+                                    # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
+                                    - 'assume:repo:${repoUrl[8:]}:branch:${short_head_branch}'
+                                else:
+                                    $if: 'tasks_for == "github-pull-request"'
                                     then:
-                                        - tc-treeherder.v2.${project}.${head_sha}
-                                        - $if: 'tasks_for == "github-push"'
-                                          then:
-                                              - index.mobile.v2.${project}.branch.${short_head_branch}.latest.taskgraph.decision
-                                              - index.mobile.v2.${project}.branch.${short_head_branch}.revision.${head_sha}.taskgraph.decision
-                                              - index.mobile.v2.${project}.revision.${head_sha}.taskgraph.decision
-                                        - $if: 'tasks_for == "cron"'
-                                          then:
-                                              # cron context provides ${head_branch} as a short one
-                                              - index.mobile.v2.${project}.branch.${head_branch}.latest.taskgraph.decision-${cron.job_name}
-                                              - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.decision-${cron.job_name}
-                                              - index.mobile.v2.${project}.branch.${head_branch}.revision.${head_sha}.taskgraph.cron.${ownTaskId}
-                          scopes:
-                              # `https://` is 8 characters so, ${repoUrl[8:]} is the repository without the protocol.
-                              $if: 'tasks_for == "github-push"'
-                              then:
-                                  $let:
-                                      short_head_branch:
```

Comment:
This let is not needed anymore because we define `short_head_branch` up above.

Reply:
This reminds me I need to redefine that in app-services too, thanks!